README.md

```diff
@@ -659,3 +662,126 @@ It is not necessary to uninstall the manually-installed version of the sonar-run
 4. Delete the SONAR_RUNNER_OPTS enviornment variable.
 5. Restart the TFS Build Service.
 	- If you have amended the environment variables then you will need to restart the Build Service so it uses the modified set of variables.
+
+
+<a name="configuringMSBuildRunnerAnchor">
+# Appendix 2: Configuring the MSBuild SonarQube Runner
+</a>
+
+## Contents
+
+- [Supplying additional analysis settings](#supplying-additional-analysis-settings)
+- [Classifying projects as test projects](#classifying-projects-as-test-projects)

```

Comment:
@duncanpMS can you remove this? We're going to update SonarLint to not trigger issues on test projects